dll/shellext/mydocs/CMyDocsDropHandler.cpp

```diff
@@ -157,7 +157,9 @@ CMyDocsDropHandler::Drop(IDataObject *pDataObject, DWORD dwKeyState,
     fileop.pFrom = pszzSrcList;
     fileop.pTo = szzDir;
     fileop.fFlags = FOF_ALLOWUNDO | FOF_FILESONLY | FOF_MULTIDESTFILES | FOF_NOCONFIRMMKDIR;
-    SHFileOperationW(&fileop);
+    hr = SHFileOperationW(&fileop);
```

Comment:
if it does not produce an HRESULT, dont call it hr.

Reply:
hr is already the type used for the return of the calling function ... which misses to fail when SHFileOperationW fails...

Reply:
SHFileOperationW does not return errors in an HRESULT format (and neither in the usual Win32 last error format). So as @learn-more hinted at, using hr (that is also used to return something from the calling function) is a mistake.